### PR TITLE
charge anhilation set Lilly (Sergio Perez and Howard Broughton)

### PR DESCRIPTION
**Pull request by sergioperezconesa**
## Input submission checklist

* [x] created a directory for each system based on the [template directory][templates]
* [x] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [x] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [x] added a PDB file with the protein named `protein.pdb`
* [x] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [x] did not re-protonate the residues, but left protonation state of the protein unchanged
* [x] kept cystallographic waters and metals in the PDB file
* [x] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [x] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Summary of changes
We followed the instructions given. In addition when two charge state were present for a ligand the neutral species was discarded and for jak1 a loop was modelled using maestro defaults.

We are happy to release these materials under the combined MIT and CCBY-SA 4.0 licenses of this repository

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/PREPARATION_DETAILS.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## System preparation details
+
+_For example:_
+
+* All prepration was carried out using maestro and text editors.
+* TPO residue was converted to THR
+* NMA was renamed NME
+* The protonated carboxilic acid was removed from the map.
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/PREPARATION_DETAILS.md:8]:
```suggestion
* The protonated carboxilic acid (ligand `39neu`) was removed from the map.
```

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/protein.pdb
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/protein.pdb:1]: I noticed that the THR residue (THR160 that used to be named TPO) still has the phosphate group. Would it be possible to convert the residue to the canonical THR?

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/itk/PREPARATION_DETAILS.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## System preparation details
+
+_For example:_
+
+* All prepration was carried out using maestro and text editors.
+* Converted N-terminal H atom to H3.
+* Converted C-terminal O atom to OXT.
+* The
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/itk/PREPARATION_DETAILS.md:8]: I think this bullet point is potentially missing the ligand prep details?

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jak1/PREPARATION_DETAILS.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  # Read in jak1
+  # Contains two PTR residues, converted to Tyr with standard Maestro mutate
+  # Ran diagnostics, has two orientations for His885.  Select default (53% occ) and commit.
+  # this His has the proton on E but is charged on D with no proton.
+  # Choose to fix by adding a hydrogen (making it a HIP really).  This maintains charge which I guess is correct
+  # Residue name was not modified.
+  # The resides from 913 to 916 are missing and the cut-off ends are capped in the input structure
+  # Same happens with 946 to 949.   Seem to be missing from multiple structures in the PDB.
+  # Obtained fasta from uniprot and added loop and sidechain building to prepwiz settings.   Obtained
+  # a structure with modeled loops (protein_loops.pdb).   Suggest running openFE on both systems, capped as provided and with loops.
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jak1/PREPARATION_DETAILS.md:11]: Yes, this sounds like a great idea!

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/thrombin/PREPARATION_DETAILS.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+_For example:_
+
+* All prepration was carried out using maestro and text editors.
+* Extracted GOL cofactor.
+* Removed PTM NAG and saturated the connected ASN.
+* Converted TYS to TYR.
+* Converted N-terminal H atom to H3 for 1C GLU (chain L), same for other chains.
+* Converted C-terminal O atom to OXT for 14K ILE (chain L), same for other chains.
+* Removed ligand
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/thrombin/PREPARATION_DETAILS.md:11]: Could you please add further details here about which ligand was removed? Thanks!

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/tyk2/PREPARATION_DETAILS.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# Preparation done in Schrodinger suite 2024-2 (patch 1)
+# import tyk2_protein.pdb into maestro.  Contains an UNK residue which turns out to be NME.   Remove cap residues
+# to allow the prepwizard capping process to work on it. Just used delete atoms, it put them back in the same place anyway.
+# Run prepwizard
+# Changed residue type of NMA to NME and number to 1178 in the gui
+# all done for tyk2
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/tyk2/PREPARATION_DETAILS.md:6]: Could you please add a bullet point describing the process for preparing the ligands (i.e. removing the neutral forms of the ligands where two protonation states were present). Thanks!

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/PREPARATION_DETAILS.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+## System preparation details
+
+_For example:_
+
+* All prepration was carried out using maestro and text editors.
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/PREPARATION_DETAILS.md:5]: Could you please add the maestro version you used during the preparation (here and in the other preparation instruction files). Thanks!

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thank you, this looks great! I just added a few comments. Please let me know if you have any questions!

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/PREPARATION_DETAILS.md (hunk 6 above)
Comment (sergioperezconesa) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/PREPARATION_DETAILS.md]: I have added this

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/thrombin/PREPARATION_DETAILS.md (hunk 4 above)
Comment (sergioperezconesa) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/thrombin/PREPARATION_DETAILS.md:11]: My bad, no ligand removed for this one

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/itk/PREPARATION_DETAILS.md (hunk 2 above)
Comment (sergioperezconesa) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/itk/PREPARATION_DETAILS.md:8]: done!

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/protein.pdb
Comment (sergioperezconesa) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/protein.pdb:1]: done!

**Comment by sergioperezconesa** [comment]
I have done all the changes and pushed them to my repo. But I am not sure if they are showing up here automatically or I have to do another PR or something.
Thanks!
Sergio

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/tyk2/PREPARATION_DETAILS.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+# removed ligand jmc_32neu to be able to have at least one charge transformation and no ligand with alternative protonation states.
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/tyk2/PREPARATION_DETAILS.md:7]:
```suggestion
# removed ligands jmc_32neu and ejm_52neu to be able to have at least one charge transformation and no ligand with alternative protonation states.
```

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thank you so much @sergioperezconesa , this looks great! I just added a small suggestion since it seemed like two ligands were removed from the original file.

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/tyk2/PREPARATION_DETAILS.md (hunk 7 above)
Comment (sergioperezconesa) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/tyk2/PREPARATION_DETAILS.md]: thanks!

**Comment by sergioperezconesa** [comment]
I think we are ready to merge

**Review by hannahbaumann — APPROVED** [review]
Thank you @sergioperezconesa , this looks good to me!

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jak1/PREPARATION_DETAILS.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+  # Preparation done in Schrodinger suite 2024-2 (patch 1)
+  # Read in jak1
+  # Contains two PTR residues, converted to Tyr with standard Maestro mutate
+  # Ran diagnostics, has two orientations for His885.  Select default (53% occ) and commit.
+  # this His has the proton on E but is charged on D with no proton.
```
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jak1/PREPARATION_DETAILS.md:5]: This is likely fine, so I don't recommend making any changes. However I'm so sorry I don't think I fully understand what is meant here, we will likely ask for further clarification when it comes to writing the manuscript.

**Review by IAlibay — APPROVED** [review]
Thanks, this looks good to me!

**Pull request merged**